src/main/java/seedu/mark/logic/commands/AddCommand.java

```diff
@@ -64,4 +65,9 @@ public boolean equals(Object other) {
                 || (other instanceof AddCommand // instanceof handles nulls
                 && toAdd.equals(((AddCommand) other).toAdd));
     }
+
+    @Override
+    public String toString() {
+        return "Add bookmark " + toAdd;
```

Comment:
```suggestion
        return "Add bookmark [" + toAdd + "]";
```

Reply:
If we're saving `Command.toString()`, can consider putting bookmark details in brackets for easy reading.